pkg/sql/copy/copy_test: TestDataDriven failed

pkg/sql/copy/copy_test.TestDataDriven [failed](https://mesolite.cluster.engflow.com/invocations/default/20eb47d9-ea1a-499b-9dec-9476257b4e56?testReportRun=9&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvc3FsL2NvcHk6Y29weV90ZXN0) on release-25.2 @ [03dcb4b5dc8e31baa16ef3d9919b1684cf2facc5](https://github.com/cockroachdb/cockroach/commits/03dcb4b5dc8e31baa16ef3d9919b1684cf2facc5):


```
          ALTER TABLE rls_table OWNER TO copier;
          ----
    copy_test.go:221: 
        /mnt/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/sql/copy/copy_test_/copy_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/copy/testdata/copy_from:776:
          exec-ddl
          SET ROLE copier
          ----
    copy_test.go:221: 
        /mnt/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/sql/copy/copy_test_/copy_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/copy/testdata/copy_from:780:
          exec-ddl
          ALTER TABLE rls_table ENABLE ROW LEVEL SECURITY, FORCE ROW LEVEL SECURITY;
          ----
    copy_test.go:221: 
        /mnt/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/sql/copy/copy_test_/copy_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/copy/testdata/copy_from:785:
          copy-from-error
          COPY rls_table FROM STDIN WITH CSV
          1,"one"
          2,"two"
          ----
          ERROR: new row violates row-level security policy for table "rls_table" (SQLSTATE 42501)
    copy_test.go:221: 
        /mnt/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/sql/copy/copy_test_/copy_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/copy/testdata/copy_from:792:
          exec-ddl
          CREATE POLICY p_ins ON rls_table FOR INSERT WITH CHECK (id > 10);
          ----
    copy_test.go:221: 
        /mnt/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/sql/copy/copy_test_/copy_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/copy/testdata/copy_from:796:
          exec-ddl
          CREATE POLICY p_sel ON rls_table FOR SELECT USING (true);
          ----
    copy_test.go:221: 
        /mnt/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/sql/copy/copy_test_/copy_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/copy/testdata/copy_from:801:
          copy-from-error
          COPY rls_table FROM STDIN WITH CSV
          20,"twenty"
          4,"four (violates rls policy)"
          ----
          ERROR: new row violates row-level security policy for table "rls_table" (SQLSTATE 42501)
    copy_test.go:221: 
        /mnt/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/sql/copy/copy_test_/copy_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/copy/testdata/copy_from:808:
          query
          SELECT * FROM rls_table ORDER BY id
        
        expected:
        
        found:
          20|twenty
    panic.go:635: -- test log scope end --
test logs left over in: outputs.zip/logTestDataDriven_vectorize=on_atomic=off_fastPath=on_copy_from2929485088
--- FAIL: TestDataDriven/vectorize=on/atomic=off/fastPath=on/copy_from (1.73s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>run=9</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDataDriven.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-54984

--- a/pkg/sql/copy/testdata/copy_from
+++ b/pkg/sql/copy/testdata/copy_from
@@ -800,8 +800,8 @@ CREATE POLICY p_sel ON rls_table FOR SELECT USING (true);
 # Deny some
 copy-from-error
 COPY rls_table FROM STDIN WITH CSV
-20,"twenty"
 4,"four (violates rls policy)"
+20,"twenty"
 ----
 ERROR: new row violates row-level security policy for table "rls_table" (SQLSTATE 42501)
 

